index.html

```diff
@@ -3,7 +3,14 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TS WEBSHOP</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
Goes for both font link tags, try to use `rel="preload"` with `as="style"` attribute. 

Difference between preload and preconnect.
```TL;DR: Resource hints are tags in HTML that help browsers load web pages faster. With them, you can speed up page loading and provide a better user experience – preload fetches and caches crucial resources, preconnect establishes early connections to external sites, and prefetch preloads likely-needed resources.```

Use case on why you should use preload: https://stackoverflow.com/questions/40624515/load-google-font-with-link-asynchronously-or-deferred-without-font-face-observ